README.md

```diff
@@ -106,13 +106,17 @@ You can run the tests with `cargo test`.
 
 ## Built With
 
-External crates :
+External crates:
 
-- `curl` for sending HTTP requests.
```

Comment:
why do you want to delete those lines (111, 112, 113) ?

Reply:
It's not an external crate, is it?

Reply:
Oh in this `Cargo.toml`: https://github.com/lethenju/spotify_rust/blob/master/spotify_api/Cargo.toml

Let me fix that then!